en/Plugins/Search.md

```diff
@@ -1,82 +1,98 @@
-![[Search.png]]
+The Search plugin helps you find files in your vault.
 
-Search is a powerful feature, and has the potential to be confusing. In most cases, if you just type what you want to find, it will work. But search has many capabilities for narrowing down to find exactly what you want.
+By default, you can find Search in the left sidebar (magnifying glass icon). You can also open Search by pressing Ctrl+Shift+F (or Cmd+Shift+F on macOS).
 
-## Quick tips
+- **Search selected text**: If you select text in the editor and open Search with the keyboard shortcut, Search shows you the search results for the selected text.
+- **Search recent search terms**: Open Search with an empty search term to list recent search terms. Click any of them to use the search term again.
 
-### Start searching
+## Search terms
 
-You can invoke search by pressing `Ctrl-Shift-F` or `Cmd-Shift-F`. You can also customize this hotkey in Settings => Hotkeys. When search is invoked, focus will be automatically put in the search bar so you can start typing your query right away.
+A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults.
 
-### Search selected text
+Each word in the search term is matched independently within each file. To search for an exact phrase, surround it with quotes, for example `"star wars"`. To search for quoted text within an exact phrase, you can _escape_ the quotes by adding a backslash (`\`) in front of the quote, for example `"they said \"hello\" to each other"`.
 
-After you select text, you can search for it by simply invoking search like above.
+You can control whether to return files that contain _all_ the words in your search term, or _any_ of the words:
 
-## Search history
+- `meeting work` returns files that contain both `meeting` and `work`.
+- `meeting OR work` returns files that contain either `meeting` or `work`.
 
-Obsidian will remember your most recently used search queries.
+You can even combine the two in the same search term.
 
-These queries will be presented to you when your search query is empty. You can click on any one of them to search for it again, allowing you to easily re-run previous queries.
+- `meeting work OR meetup personal` returns files for work meetings and personal meetups.
```

Comment:
Did you reverse the order of keywords to signify the order of keywords within a document doesn't matter?

Reply:
That was the idea at least :) Maybe we can find a better example if it's too contrived.